src/AAVE/Modal/WithdrawModal.jsx

```diff
@@ -501,12 +505,22 @@ return (
                     .toFixed(0);
                   approveForGateway(aTokenAddress, amount)
                     .then((tx) => {
-                      tx.wait().then((res) => {
-                        const { status } = res;
-                        if (status === 1) {
-                          State.update({ needApprove: false, loading: false });
-                        }
-                      });
+                      tx.wait()
+                        .then((res) => {
+                          const { status } = res;
+                          if (status === 1) {
+                            State.update({
+                              needApprove: false,
+                              loading: false,
+                            });
```

Comment:
maybe also add the `console.log("tx succeeded", res);` logging here. 

make it consistent with other places and easier for debugging.